ci/install.sh

```diff
@@ -45,11 +45,23 @@ EOF
     fi
 }
 
+install_zsh() {
+    case $TARGET in
+        aarch64-unknown-linux-gnu)
```

Comment:
I don't think I understand this. Why are you trying to install zsh only when testing on aarch64? aarch64 is ARM. I suspect ripgrep probably works on ARM, but I don't run CI on it.